tools/cloudera/build_hue_common.sh

```diff
@@ -308,8 +321,8 @@ function ubuntu18_install() {
         zlibc'
     # MySQLdb install
     # It is pre-installed
-    # NODEJS 14 install
-    sudo -- sh -c 'curl -sL https://deb.nodesource.com/setup_14.x | sudo bash - && \
+    # NODEJS 16 install
```

Comment:
Update comment to say why we use v 16

Reply:
Added a comment to explain why we are moving to v16.